charts/corda-dev/kafka-config/server.properties

```diff
@@ -0,0 +1,24 @@
+listeners=SASL_SSL://:9092,CONTROLLER://:9093,SASL_PLAINTEXT://:9094
```

Comment:
Why is this file needed?

Reply:
I started out just specifying command line overrides for the Kafka options that I wanted to change but ended up overriding nearly all of them so it was just simpler to provide the entire config file to Kafka.